flax/core/scope.py

```diff
@@ -1074,9 +1074,19 @@ def apply(
   def wrapper(
     variables: VariableDict,
     *args,
-    rngs: Optional[RNGSequences] = None,
+    rngs: Optional[Union[PRNGKey, RNGSequences]] = None,
```

Comment:
```suggestion
    rngs: PRNGKey | RNGSequences | None = None,
```

Reply:
won't this fail Github CI for python 3.9?

Reply:
our minimum Python version is still [3.9](https://github.com/google/flax/blob/main/pyproject.toml#L7).